tests/system/action/user/test_set_password_self.py

```diff
@@ -42,35 +41,41 @@ def test_set_password_self_anonymus(self) -> None:
         self.assert_status_code(response, 400)
         self.assertIn("Can't set password for anonymous", response.json["message"])
 
-    # TODO: fix when permission is correctly implemented
-    @pytest.mark.skip()
-    def test_set_password_self_temporary_permissions(self) -> None:
+    def test_set_password_self_permissions(self) -> None:
```

Comment:
I think this test is no longer needed since it is the same as the one above.

Reply:
Removed.